python/sglang/srt/entrypoints/http_server.py

```diff
@@ -124,7 +124,7 @@ async def lifespan(fast_api_app: FastAPI):
 
 
 # Fast API
-app = FastAPI(lifespan=lifespan)
+app = FastAPI(lifespan=lifespan, openapi_url=None if os.environ.get("DISABLE_OPENAPI_DOC") == "true" else "/openapi.json")
```

Comment:
1. Please fix the lint.
2. Please use https://github.com/sgl-project/sglang/blob/9b8333d99204ea8582d5132dd48084decaa68d25/python/sglang/srt/utils.py#L1266

Reply:
Thanks, done